apis/field_error.go

```diff
@@ -251,6 +256,16 @@ func (fe *FieldError) normalized() []*FieldError {
 	return errors
 }
 
+// WrappedErrors returns the value of the nested errors.
+func (fe *FieldError) WrappedErrors() []FieldError {
+	if fe == nil {
+		return nil
+	}
+	results := make([]FieldError, 0, len(fe.errors))
+	results = append(results, fe.errors...)
```

Comment:
Are you sure the errors here are already flattened? 🤔

Reply:
They are not flattened. That is done in `normalized()`. I've added a change to expose that as well so that we can access in a flattened way.

Reply:
I think we may just want to call `normalize()` and possibly return that? 🤔

Reply:
Make sense. I've changed `WrappedErrors = merge(normalized())`.